I.write-tests/02-remove-vowels/remove-vowels.test.js

```diff
@@ -1,8 +1,14 @@
-test('remove vowels from word', function () {
+test("remove vowels from word", function() {
+  // Arrange
+  // Given
+  var removeVowels = require("./remove-vowels.js");
+  var word = "Seraphine";
 
-    // Arrange
+  // Act
+  //When
+  var result = removeVowels(word);
 
-    // Act
-
-    // Assert
-})
\ No newline at end of file
+  // Assert
+  //Then
+  expect(result).toEqual("_e_a__i_e");
```

Comment:
Actually given the name of the function is called "removeVowels", we can clearly tell there's a bug! The test would expect "Seraphine" to result in "S_r_ph_n_" instead. It will fail with current code, so then you can fix the function till it passes